ee-security/README.adoc

```diff
@@ -1,31 +1,29 @@
 include::../shared-doc/attributes.adoc[]
 
-= ee-security: Using EE Security with WildFly Elytron
+= ee-security: Using Jakarta EE Security
 :author: Darran Lofthouse
 :level: Intermediate
-:technologies: EE Security, Servlet, CDI, WildFly Elytron
+:technologies: EE Security, Servlet, CDI
 
 [abstract]
-The `ee-security` quickstart demonstrates how EE security can be used with WildFly Elytron.
+The `ee-security` quickstart demonstrates how Jakarta EE security.
```

Comment:
Just a minor comment, the word "how" should now be removed .

Reply:
@fjuma done